diff --git a/bin/IndexGenerator.php b/bin/IndexGenerator.php
index 69a303e..c5d5e47 100644
--- a/bin/IndexGenerator.php
+++ b/bin/IndexGenerator.php
@@ -450,7 +450,14 @@ class IndexGenerator
                 //|| preg_match('/DateTime/', $file) //zend
                 || preg_match('/DateTimeSelect/', $file) //zend
                 || preg_match('/MonthSelect/', $file) //zend
-                //|| preg_match('/PropelDataCollector/', $file) //zend
+
+                || preg_match('/ComplexService.php/', $file) //custom
+                || preg_match('/\/Tests\//', $file) //custom
+                || preg_match('/Amqp/', $file) //custom
+                || preg_match('/SolrPoolingClient/', $file) //custom
+                || preg_match('/PdependExtension/', $file) //custom
+                || preg_match('/Validator/', $file) //custom
+                //|| preg_match('/PropelDataCollector/', $file) //custom
             ){
                 continue;
             }
